tests/markdown-extended.md

```diff
@@ -89,9 +89,9 @@ That is so funny! :joy:
 
 :cowboy_hat_face::nerd_face:
 
-## Highlight
+## Highlight/Mark
```

Comment:
Sure, don't mind things like these

Another one that has 2 different names is Definition list/Description list but I thought using only the first was okay in the context

Reply:
Yes I think it's fine. I just added it here because "mark" and "highlight" seem quite different ^^ definition vs. description list at least sounds like it could be the same thing